### ISSUE TITLE
Canada.ca menu examples issuesC

### ISSUE THREAD

**Issue by dfarough** [wet-boew/GCWeb#1458]
I have looked at the Canada.ca examples paying particular attention to the menus.

AS a person having low vision (5% vision one eye) I believe that the focus styles in the menus on the right side of the screen are too subtle for someone with my degree of vision to easily perceive.
The outline around the chosen option is barely visible in internet explorer. I feel that a different background colour for the focussed item would improve this design.
Also when moving from the right side of the screen back to the menu that previously had focus, it is difficult to perceive any change or difference from before the item lost focus.  I am wondering whether a different style should be used when focus moves to items within the menu.  That would make it more obvious when focus moves back to the menus on the left side of the screen.

Although I primarily use a screen reader to access content, I always need to know where the cursor is.  It is not enough for me to hear the screen reader tell me where the focus is, I will always try to verify this visually.

I believe that this two column menu design may be confusing for people, given that no indication is provided that using the right arrow will take you to most requested options.  One might argue that perhaps most requested options should come at the top of the menu structure.  It also seems odd to have Identical content at the beginning of the menu options on the right hand side of the screen.  It requires an extra keystroke before getting to the other options in the menu.  This may make this menu structure confusing for those who cannot see what is happening on screen.
For what it’s worth, to me our menu bar option that goes horizontally across the top of the page makes much more sense and is easier for users to navigate.  I find that arrowing through the main menu options is a bit sluggish because of the need to display the available options in the right hand column.

I also notice that Jaws will prompt the user to use the Jaws Key + alt+shift + m to move to the menu options for the menu items in the menu options.  However when trying this key combination Jaws says “Failed to move to element”.  Jaws will provide this prompt when an item contains the ARIA-controls attribute. This is too verbose and of course is not relevant since using this keystroke does not work as advertised.

Another little thing that I’ll include is the following:

When using the English example page, when I activate the Basic html option. The French Menu items are displayed.

Also once in basic html mode the menus will not expand.

One other thing I am wondering about!

Given all of the vertical space that these menus use, I am wondering if it would help for mobile users to have a collapse menu option at the end of the menu selections.  This way you don’t have to swipe all the way to the top to collapse the menu.  I have to confess that I don’t know all of the gestures very well, so there might be a way to do this that I am not aware  of.
I really do think that the most requested sub menu should appear first in the menu structure so that users can easily find the most popular items.

**Comment by duboisp** [comment]
Thanks for the great feedback

**Comment by patlaj** [comment]
Yes just to echo @duboisp, we really appreciate you taking the time to lay out all of these issues. It's already led to productive discussions and we'll definitely be making updates to address them.

**Comment by duboisp** [comment]
@dfarough did you tested only with one combination of screen reader, browser and view port? Like Jaws and IE11 in the mobile view? Thank you.
